### PR TITLE
Register function

### PR DESCRIPTION
**Pull request by SChaudhary99**
Guys the Login page is Created and its working, and Registration page is working as well.
Please, review the code and let me know if i have to update or change anything.
Database platform mySQL workbench.

### CODE REVIEW

File: src/main/java/com/example/job_service/config/SecurityConfig.java [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+    @Autowired
+    CustomUserDetailService customUserDetailService;
+
+    @Bean
+    public SecurityFilterChain filterChain(HttpSecurity http) throws Exception {
+
+        return http.csrf(AbstractHttpConfigurer::disable)
+                .authorizeHttpRequests(auth -> auth
+                        .requestMatchers("/", "/register","/login").permitAll()
+                        .requestMatchers("/admin").hasRole("USER")
```
Comment (nitika807) [inline src/main/java/com/example/job_service/config/SecurityConfig.java:32]: is this page for user or the admin?

File: src/main/java/com/example/job_service/config/SecurityConfig.java [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+                        .anyRequest()
+                        .authenticated()
+                )
+                .formLogin(login -> login
+                        .loginPage("/login").permitAll()
+                                .failureUrl("/login?error=ture")
```
Comment (nitika807) [inline src/main/java/com/example/job_service/config/SecurityConfig.java:38]: spelling mistake

**Review by nitika807 — APPROVED** [review]
the code seems fine to me for the initial stage. It is working fine although the user's first/last name is accepting numeric data. I have written my comments for the area of concerns.

File: src/main/java/com/example/job_service/config/SecurityConfig.java (hunk 1 above)
Comment (SChaudhary99) [reply src/main/java/com/example/job_service/config/SecurityConfig.java:32]: it’s for user home page after login, i mean i created it for admin use but forgot to rename it

File: src/main/java/com/example/job_service/config/SecurityConfig.java (hunk 2 above)
Comment (SChaudhary99) [reply src/main/java/com/example/job_service/config/SecurityConfig.java:38]: sorry, which spelling mistake?

Comment (nitika807) [reply src/main/java/com/example/job_service/config/SecurityConfig.java:38]: Shouldn’t it be error =true?

Comment (SChaudhary99) [reply src/main/java/com/example/job_service/config/SecurityConfig.java:38]: oh no no it’s called ternary operator i don’t exactly know if i have used correctly or not but i think it basically means if error = true then redirect to login page, (? mean if the ?’s right side is true then do the ?’s left side part)

**Review by navtej-singh — COMMENTED** [review]
It looks good, will test it.

File: src/main/java/com/example/job_service/config/SecurityConfig.java (hunk 2 above)
Comment (nitika807) [reply src/main/java/com/example/job_service/config/SecurityConfig.java:38]: Yes that I understood already. I wasn’t sure if “ture” is a keyword.

**Pull request closed without merging**